aemedge/blocks/table/table.css

```diff
@@ -1,51 +1,64 @@
 .table {
-    width: 100%;
-    overflow-x: auto;
-  }
-  
-  .table table {
-    width: 100%;
-    max-width: 100%;
-    border-collapse: collapse;
-    font-size: var(--udexTypographyBodySFontSize);
-  }
-  
-  .table table thead tr,
-  .table table tbody tr{
-    border-bottom: var(--sapList_BorderWidth) solid var(--sapList_GroupHeaderBorderColor);
-  }
-  
-  .table table tbody tr {
-    background-color: var(--sapList_GroupHeaderBackground);
-  }
-  
-  .table table th {
-    color: var(--sapList_HeaderTextColor);
-    font-weight: var(--udexTypographyFontWeightBold);
-  }
-  
-  .table table th,
-  .table table td {
-    padding: 8px;
-    text-align: left;
-
-    & p {
-      margin: 0;
-    }
-  }
-  
-  /* no header variant */
-  .table.no-header table tbody tr {
-    border-top: var(--sapList_BorderWidth) solid var(--sapList_GroupHeaderBorderColor);
-  }
-  
-  /* striped variant */
-  .table.striped tbody tr:nth-child(odd) {
-    background-color: var(--sapList_GroupAlternatingHeaderBackground);
+  width: 100%;
+  overflow-x: auto;
+  background-color: var(--sapList_TableGroupHeaderBackground);
+  color: var(--sapList_TableGroupHeaderTextColor);
+}
+
+.table table {
+  width: 100%;
+  max-width: 100%;
```

Comment:
Why we need max-width when we already set width?

Reply:
I am not sure, again this is I think copied across from the [example block collection](https://github.com/adobe/aem-block-collection/blob/main/blocks/table/table.css). It's also a bit strange because I thought that max-width only applies to block elements. I am assuming another workaround for a particular browser?